open_spiel/python/mfg/algorithms/dqn_policies.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019 DeepMind Technologies Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Computes a set of DQN policies for a list of agents and environments."""
+import numpy as np
+
+from open_spiel.python import policy as policy_std
+from open_spiel.python import rl_environment
+from open_spiel.python.jax import dqn
+
+class DQNPolicies(policy_std.Policy):
+  """Joint policy to be evaluated."""
```

Comment:
We can perhaps use a more general description here, e.g. a joint policy of DQN policies.